crate/server/src/database/redis/datasets.rs

```diff
@@ -0,0 +1,88 @@
+use std::collections::HashMap;
+
+use async_trait::async_trait;
+use cosmian_findex_structs::{EncryptedEntries, Uuids};
+use redis::pipe;
+use tracing::{instrument, trace};
+use uuid::Uuid;
+
+use super::Redis;
+use crate::{
+    database::database_traits::DatasetsTrait,
+    error::{result::FResult, server::FindexServerError},
+};
+
+/// Generate a key for the dataset table
+fn build_dataset_key(index_id: &Uuid, uid: &Uuid) -> Vec<u8> {
```

Comment:
What does the second `u` of UUID stand for ? I thought it was `User`.

Reply:
It stands for Universally Unique IDentier. https://crates.io/crates/uuid